kubernetes/template-generator/cli.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import re
+
+from template_generator import MultivolumeTemplateGenerator
+from template_generator import MacroimageTemplateGenerator
+from template_generator import TemplateGenerator
+
+from container_os import RHEL7ContainerOS
+from container_os import RHEL6ContainerOS
+
+
+LEAPP_BASEPATH = "/var/lib/leapp/macrocontainers"
```

Comment:
This would make sense if we have had this in leapp tool, since we do not have it there yet, I do not think it is necessary to implement it at this point. But good point for future planning.